.github/workflows/ci.yml

```diff
@@ -2,37 +2,30 @@ name: CI
 
 on:
   pull_request:
-  schedule:
-    - cron: '4 4 * * *'
 
 jobs:
   setup_matrix:
-    if: github.event_name != 'schedule' || github.repository_owner == 'theforeman'
-    name: 'Setup Test Matrix'
+    name: Static validations
     runs-on: ubuntu-latest
     outputs:
       beaker_setfiles: ${{ steps.get_outputs.outputs.beaker_setfiles }}
       puppet_major_versions: ${{ steps.get_outputs.outputs.puppet_major_versions }}
```

Comment:
I think these were dropped in the latest puppet_metadata, so I need to replace this. Perhaps I need to solve the beaker facts properly instead.